System.IO.Abstractions.TestingHelpers/IMockFileDataAccessor.cs

```diff
@@ -62,5 +62,7 @@ public interface IMockFileDataAccessor
         IDriveInfoFactory DriveInfo { get; }
 
         PathVerifier PathVerifier { get; }
+
+        IFileSystem FileSystem { get; }
```

Comment:
This feels slightly icky to me, but it was the least invasive change to make, as the various constructors for all the mock classes can then remain the same.

Reply:
I like it this way!